client/src/components/Ticketing/ticketingmanager/Season/components/SeasonEvents.tsx

```diff
@@ -1,7 +1,229 @@
+    // Fetch all event API returns data with different property names than what update event API requires as payload
+    const {
+      id: eventId,
+      description: eventDescription,
+      title: eventName,
+    } = eventToAdd;
+    eventToAdd = {
+      ...eventToAdd,
+      eventid: eventId,
+      eventname: eventName,
+      eventdescription: eventDescription,
+      seasonid_fk: seasonId,
+    };
+    delete eventToAdd['id'];
+    delete eventToAdd['seasonid'];
+    delete eventToAdd['seasonticketeligible'];
+    delete eventToAdd['numshows'];
+    delete eventToAdd['description'];
+    delete eventToAdd['title'];
+
+    const updateEventCall = await updateEventSeason(eventToAdd, token);
+    setShowPopUp(true);
+    if (updateEventCall) {
+      setAllEventInfo([...allEventInfo, eventToAddCopy]);
+      setEventsNotInSeason(updatedEventsNotInSeason);
+      setPopUpMessage({
+        title: 'Success',
+        message: 'The event has been added to the season!',
+        success: true,
+        handleClose: () => setShowPopUp(false),
+        handleProceed: () => setShowPopUp(false),
+      });
+    } else {
+      setPopUpMessage({
+        title: 'Error',
+        message:
+          'There was a error adding the event to the season. Pleas try again.',
+        success: false,
+        handleClose: () => setShowPopUp(false),
+        handleProceed: () => setShowPopUp(false),
+      });
+    }
+  };
+
+  useEffect(() => {
+    void handleGetAllEvents();
+  }, []);
+
+  return (
+    <div className='rounded-xl p-7 bg-white text-lg mt-5'>
+      <section className='flex flex-col gap-4 items-center mb-6 tab:flex-row tab:justify-center tab:flex-wrap min-[1076px]:justify-between'>
+        <article className='flex flex-wrap gap-2'>
+          <h1 className='text-3xl'>Season Events </h1>
+          <button
+            className='flex gap-1 items-center bg-green-500 hover:bg-green-700 disabled:bg-gray-500 text-white p-2 rounded-xl'
+            disabled={isFormEditing || isAddEventActive}
+            onClick={() => setIsAddEventActive(true)}
+          >
+            <svg
+              xmlns='http://www.w3.org/2000/svg'
+              fill='none'
+              viewBox='0 0 20 20'
+              strokeWidth='3'
+              stroke='currentColor'
+              className='w-5 h-5 mb-1'
+            >
+              <path
+                strokeLinecap='round'
+                strokeLinejoin='round'
+                d='M12 6v12m6-6H6'
+              />
+            </svg>
+            <p>Add event</p>
+          </button>
+        </article>
+        <article>
+          <label className='pr-2' htmlFor='eventSelect'>
+            Sort by:
+          </label>
+          <select
+            name='sortBy'
+            id='eventSelect'
+            className='p-1 border border-zinc-500 rounded-lg'
+          >
+            <option value='eventName'>Event Name - Ascending</option>
+          </select>
+        </article>
```

Comment:
I included it to be consistent with the events page but I don't know if we should have a select option for sorting on the seasons page. I can only really think of sorting by event name. Sorting by event date or time doesn't really make sense because the date/time aren't being displayed for each event in so there isn't much to do with that information. Would you prefer having a select or removing it entirely?

Reply:
Yeah honestly, I think we could probably remove this. They would really only have at most a couple events per season so they should be able to easily find the events they're looking for without the `select`